src/blocks/product_card.css

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  width: 200px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 6px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__img {
+  display: block;
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: 160px 134px;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto;
+}
+
+.product-card__name {
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+  margin-top: 40px;
+  font-size: 12px;
+}
+
+.product-card__code {
+  color: #616070;
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.product-card__rate {
+  margin-top: 17px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-card__reviews {
+  display: flex;
+  align-items: center;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.product-card__price {
+  margin-top: 23px;
+  height: fit-content;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-card__label {
+  color: #616070;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.product-card__value {
+  color: #060b35;
+  font-weight: 700;
+}
+
+.product-card__buy-button {
+  display: block;
+  margin: 16px 0 auto;
+  text-align: center;
+  padding: 10px 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  width: 100%;
```

Comment:
do we need 100% for a block element?

Reply:
No, because it would  be as wide as parent element is (100% from it). Maybe I forgot to delete it, when my element was inline-block or smth like that